test/subscription.ts

```diff
@@ -330,18 +330,19 @@ describe('Subscription', () => {
   describe('close', () => {
     it('should call the success callback', done => {
       sandbox.stub(subscriber, 'close').resolves();
-      subscription.close(done);
+      subscription.close?.(done);
     });
 
     it('should pass back any errors that occurs', done => {
       const fakeErr = new Error('err');
 
       sandbox.stub(subscriber, 'close').rejects(fakeErr);
 
-      subscription.close(err => {
-        assert.strictEqual(err, fakeErr);
-        done();
-      });
+      subscription?.close!(),
```

Comment:
I think for whatever reason, the `?.` here is failing, so the test never concludes. I'm unsure why it would be undefined here and not before, but it's probably a more discoverable test failure if it has an undefined object exception anyhow (i.e. I'd just switch `subscription` back to `subscription!` for the Partial).